tests/micro/arduino/conftest.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import pathlib
+
+import pytest
+
+import tvm.target.target
+
+# The models that should pass this configuration. Maps a short, identifying platform string to
+# (model, zephyr_board).
+PLATFORMS = {
+    "due": ("sam3x8e", "due"),
+    "feathers2": ("esp32", "feathers2"),
+    "nano33ble": ("nrf52840", "nano33ble"),
+    "pybadge": ("atsamd51", "pybadge"),
+    "spresense": ("cxd5602gg", "spresense"),
+    "teensy40": ("imxrt1060", "teensy40"),
+    "teensy41": ("imxrt1060", "teensy41"),
+}
+
+TEMPLATE_PROJECT_DIR = (
+    pathlib.Path(__file__).parent
+    / ".."
+    / ".."
+    / ".."
+    / "apps"
+    / "microtvm"
+    / "arduino"
+    / "template_project"
+).resolve()
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--microtvm-platforms",
+        default=["due"],
```

Comment:
for checkin, should avoid preferring any one hardware platform (a software-only one is ok, since that makes the test always run out of the box; a hardware one should require manual confirmation from the user that this is the board they've connected)